amazon-cloudwatch-agent-operator.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: amazon-cloudwatch-agent-operator
+  version: 1.6.0
+  epoch: 0
+  description: Software developed to manage the CloudWatch Agent on kubernetes.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: e4fd9a62a095b26e58fdc09cc59a0e9f10b0e333
+      repository: https://github.com/aws/amazon-cloudwatch-agent-operator
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0 github.com/hashicorp/go-retryablehttp@v0.7.7 github.com/Azure/azure-sdk-for-go/sdk/azidentity@v1.6.0 github.com/docker/docker@v25.0.6
+
+  - uses: go/build
+    with:
+      output: manager
+      packages: .
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr/bin
+          ln -sf /manager ${{targets.contextdir}}/usr/bin/manager
+    description: Compatibility symlink for ${{package.name}}
+
```

Comment:
We should add a test here

Reply:
tried to add tests but failed for both version and help flags.